packages/binding-mqtt/README.md

```diff
@@ -1,5 +1,13 @@
 # Eclipse Thingweb node-wot MQTT Binding
```

Comment:
This title here is a little bit different from the others, should we align it?

Reply:
Fixed it now in the last commit